lib/game_statistics.rb

```diff
@@ -4,4 +4,11 @@ def initialize(game_stats)
         @game_stats_data = game_stats # hashes of game objects
     end
 
+    def highest_total_score
+        highest_scoring_game = @game_stats_data.max_by do |game_id, game_object|
+            game_object.total_goals
+        end
+        highest_scoring_game[1].total_goals
```

Comment:
Because `highest_scoring_game` is an array, we reference the "value" by index location, as opposed to referencing a hash's key to return the value